### PR TITLE
Update stories.md

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

**Comment by Luyao-Zhang-1** [comment]
@Adam-Dupaski  could you please take a look?

File: content/developerportal/collaborate/stories.md [hunk 1]
```diff
@@ -228,35 +228,34 @@ Moving a story will move the tasks as well.
 
 #### 2.6.3 Adding a Task to a Story {#adding-task}
 
-To add a task to a story, access the [story details](#story-details) page and under **Tasks**, start typing in the **Add a new task** box.
+To add a task to a story, access the [Story Details](#story-details) page and under **Tasks**, start typing in the **Add a new task** box.
```
Comment (Adam-Dupaski) [inline content/developerportal/collaborate/stories.md:231]: @Luyao-Zhang-1 capitalizing "story details" is not necessary, as that is not a separate doc, nor a section name, nor does it appear in the UI as capitalized. This should also be changed above in section 2.6.1.

**Review by Adam-Dupaski — APPROVED** [review]

File: content/developerportal/collaborate/stories.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/developerportal/collaborate/stories.md:231]: @Adam-Dupaski Good one. I will fix it.

Comment (Adam-Dupaski) [reply content/developerportal/collaborate/stories.md:231]: @Luyao-Zhang-1 besides this, looks good, ready to merge, thanks!

**Pull request merged**